acid/features/history/tests/conftest.py

```diff
@@ -8,16 +8,27 @@
 from ..model import ZuulBuild, ZuulBuildSet
 
 
-@pytest.fixture
-def zuul_builds(mocker):
-    mocker.patch('pony.orm.core.commit')
-    return [make_build(build_number=104, buildset_id=5010),
-            make_build(build_number=104, buildset_id=5010)]
+@db_session
+def make_build(build_number=104, buildset_id=5010, start_time=0, end_time=0):
+    if (start_time == 0):
+        start_time = datetime(2018, 2, 23, 22, 0, 0)
+    if (end_time == 0):
+        end_time = datetime(2018, 2, 23, 23, 55, 0)
+    return ZuulBuild(buildset_id=buildset_id,
+                     uuid='86394a77c99f45aba0e299b660214a9c',
+                     job_name='acid-build-nightly',
+                     result='SUCCESS',
+                     start_time=start_time,
+                     end_time=end_time,
+                     voting=True,
+                     log_url=f'http://logs.acid.test/{build_number}/865548a7/',
+                     node_name='first_node')
 
 
 @pytest.fixture
 def make_buildset():
-    def _make_buildset(build_number, branch='master'):
+    def _make_buildset(build_number=104, branch='master', number_of_builds=5,
+                       start_time=0, end_time=0):
```

Comment:
I think a `None` is more common value for default, especially if you expect something as complex as datetime object. The `0` value indicates that you expect a number.